unity-renderer/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -310,15 +310,12 @@ public IEnumerator LoadScene(int sceneIndex = -1, bool showSceneObj = true, Acti
 
                 if (matTransitions != null && matTransitions.Length > 0)
                 {
-                    float bailout = 0;
                     //NOTE(Brian): Wait for the MaterialTransition to finish before copying the object to the library
                     yield return new WaitUntil(
                         () =>
                         {
-                            bailout += Time.deltaTime;
-
-                            if (bailout > 5.0f)
-                                return true;
+                            // IMPORTANT: We shouldn't bail out of this waiting, as that will provoke the asset saved in
+                            // the library to have the hologram material and trigger further problems
```

Comment:
Great job! I would wait for Brian's approval to understand the reasoning behind the the bailout, but looks good to me.

Reply:
As talked with Brian in the meeting with the 3 of us, I'm moving forward with merging this PR.